charts/camunda-platform/charts/tasklist/templates/configmap.yaml

```diff
@@ -21,10 +21,14 @@ data:
         host: {{ .Values.global.elasticsearch.host }}
         # Transport port
         port: {{ .Values.global.elasticsearch.port }}
+        {{- if .Values.global.elasticsearch.url }}
+        # Elasticsearch full url
+        url: {{ .Values.global.elasticsearch.url }}
+        {{- end }}
       # Zeebe instance
       zeebe:
         # Broker contact point
-        brokerContactPoint: {{ tpl .Values.global.zeebeClusterName . }}-gateway:26500
+        brokerContactPoint: "{{ tpl .Values.global.zeebeClusterName . }}-gateway:{{ .Values.global.zeebePort }}"
```

Comment:
:thought_balloon: Good catch! Maybe for next time might be worth for a separate commit, since it fixes something different.

Reply:
I've split it into a separated commit :heavy_check_mark: